name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/PDFInputFile.java

```diff
@@ -73,12 +74,22 @@ public void convertPDFtoText() throws IOException
     {
         try (PDDocument document = PDDocument.load(getFile()))
         {
+            boolean isProtected = document.isEncrypted();
+            if (isProtected)
+            {
+                document.decrypt(""); //$NON-NLS-1$
+                document.setAllSecurityToBeRemoved(true);
+            }
             PDDocumentInformation pdd = document.getDocumentInformation();
             author = pdd.getAuthor() == null ? "" : pdd.getAuthor(); //$NON-NLS-1$
 
             PDFTextStripper textStripper = new PDFTextStripper();
             textStripper.setSortByPosition(true);
             text = textStripper.getText(document);
         }
+        catch (CryptographyException e)
+        {
+            // TODO Auto-generated catch block
```

Comment:
Let's re-package this as IOException and throw it so that we can inform the user.